angular.json

```diff
@@ -23,7 +23,24 @@
               "src/assets"
             ],
             "styles": [
-              "src/styles.css",
+              "src/styles/styles.css",
+              "src/styles/pagination.css",
+              "src/styles/slidertoggle.css",
```

Comment:
let's use camel casing to name these files. "sliderToggle.css" "profilePicture.css" etc..